vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -190,9 +190,9 @@ for example, they can run the following:
 ```{r, eval = FALSE}
 run_stress_test(
   asset_type = "loans",
-  input_path_project_specific = "/path/to/specific/data/inputs",
-  input_path_project_agnostic = "/path/to/agnostic/data",
-  output_path = "/path/to/output/directory",
+  input_path_project_specific = "/example_project/project_specific_input/",
+  input_path_project_agnostic = "/example_project/project_agnostic_input/",
+  output_path = "/example_project/output",
   shock_year = c(2025, 2028, 2032, 2035)
 )
 ```
```

Comment:
when searching the repo for "/path/to/" one can see that in the readme and in vignette 3 there are still mentions of the old file name convention. Please adjust at least in vignette 3. If you want to also adjust it in the readme please adjust only the .md file, the rmd file will be rendered by a GH action.